nxdrive/direct_edit.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, manager: "Manager", folder: Path) -> None:
         self.use_autolock = self._manager.get_direct_edit_auto_lock()
         self._event_handler: Optional[DriveFSEventHandler] = None
         self._metrics = {"edit_files": 0}
+        self._opened_files = {}
```

Comment:
You should use a `set`. Then, you can use `.discard()` and `.add()` in other places.